Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:1.17
+
+WORKDIR /go/src/walrus
+
+COPY . .
+
+EXPOSE 3000
+
+CMD ["make"]
```

Comment:
I think it would be better to use `ENTRYPOINT` keyword here. You just need to compile the application with go build and pass the binary's path. This `EXPOSE` is not needed as well.

Reply:
Done.